packages/core/database/src/metadata/metadata.ts

```diff
@@ -41,11 +41,13 @@ export class Metadata extends Map<string, Meta> {
     }
   }
 
-  loadModels(models: Model[] = []) {
+  loadModels(models: Model[], options: MetadataOptions) {
     // init pass
-    for (const model of _.cloneDeep(models)) {
+    for (const model of cloneDeep(models ?? [])) {
```

Comment:
Do we still have the issue where the modifying the indexes in the metadata will also update the indexes in the models because of how we're dealing with the referenced array? Or was it coming from somewhere else?

Reply:
The issue was either caused by something I did, or (my suspicion) is actually still an issue but doesn't affect the migration